Upstream always installs all available localization files. We workaround it by
sed'ing out unneeded languages from PO variable in aegisub/po/Makefile file.
This can lead to an empty PO definition and in this case install target fails.
Thus a simple guard is introduced to prevent this fail.

diff --git a/aegisub/po/Makefile b/aegisub/po/Makefile
index 0b73bde..957c123 100644
--- a/aegisub/po/Makefile
+++ b/aegisub/po/Makefile
@@ -38,11 +38,13 @@ all: $(MO)
 	$(BIN_MSGFMT) -o $@ $<
 
 install:
+ifdef PO
 	@$(BIN_MKDIR) -p $(addsuffix /LC_MESSAGES/, $(addprefix $(DESTDIR)$(P_LOCALE)/, $(basename $(PO))))
 	@for i in $(basename $(PO)); do \
 		echo $(BIN_INSTALL) $$i.mo $(DESTDIR)$(P_LOCALE)/$$i/LC_MESSAGES/$(AEGISUB_CATALOG).mo; \
 		$(BIN_INSTALL) $$i.mo $(DESTDIR)$(P_LOCALE)/$$i/LC_MESSAGES/$(AEGISUB_CATALOG).mo; \
 	done
+endif
 
 
 CLEANFILES = $(MO)
